lib/utils/BaseReporter.js

```diff
@@ -217,7 +217,7 @@ class BaseReporter extends events.EventEmitter {
                       this.color('error message', '%s') +
                       this.color('bright yellow', '%s') +
                       this.color('error stack', '\n%s\n')
-            console.log(fmt, (i + 1), test.title, test.err.message, test.runningBrowser, test.err.stack)
+            console.log(fmt, (i + 1), test.parent + ': ' + test.title, test.err.message, test.runningBrowser, test.err.stack)
```

Comment:
I mean that you should care about block levels.:

Actual:

```
describe
title
```

Expected:

```
describe
   title
```

![reporter-spec-fail](https://cloud.githubusercontent.com/assets/803674/16537465/9c454cf8-400b-11e6-946e-fbf84c3b0512.png)